subsys/bluetooth/host/conn_internal.h

```diff
@@ -208,7 +208,7 @@ struct bt_conn_tx {
 	void *user_data;
 };
 
-struct acl_data {
+struct bt_conn_rx {
```

Comment:
```suggestion
struct bt_conn_rx_user_data {
```

Reply:
Since `bt_conn` no longer (after ISO) exclusively refer to ACL, I think it makes sense to use the term ACL for structs specific for ACL connections